pkgs/development/tools/mold/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv, fetchFromGitHub, lib, clang_12, cmake, lld_12, tbb, xxHash, zlib, openssl, git }:
+stdenv.mkDerivation rec {
+  pname = "mold";
+  version = "0.9.1";
+  src = fetchFromGitHub {
+    owner = "rui314";
+    repo = "mold";
+    rev = "v${version}";
+    sha256 = "sha256-yIkW6OCXhlHZ1jC8/yMAdJbSgY9K40POT2zWv6wYr5E=";
+  };
+  nativeBuildInputs = [ clang_12 cmake lld_12 tbb xxHash zlib openssl git ];
+  dontUseCmakeConfigure = "true";
```

This line looks weird like this.
```suggestion
  dontUseCmakeConfigure = true;
```